app/search_builders/curation_concerns/filter_by_type.rb

```diff
@@ -3,7 +3,7 @@ module FilterByType
     extend ActiveSupport::Concern
 
     included do
-      self.default_processor_chain += [:filter_models]
+      self.default_processor_chain += [:filter_models, :only_active_works]
```

Comment:
@jcoyne: @straleyb is addressing this change in his WIP PR https://github.com/projecthydra/curation_concerns/pull/1085 He is calling the module `FilterSuppressed`